app/views/nsm/work_items/confirm_delete_adjustment.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= title t('.page_title') %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= t('.page_title') %></h1>
+    <p>
+    <%= t('.cost_type', cost_type: @adjustment.work_type) %>
+    <br>
+    <%= t('.reason', reason: @adjustment.reason) %>
+    <br>
+    <%= raw t('.time_allowed', time_allowed: @adjustment.formatted_allowed_time_spent) %>
```

Comment:
_In general_ we want to avoid `raw` as it's overly permissive. If we change the string from `time_allowed` to `time_allowed_html` to use I18n.t's magic sanitisation logic, can we remove `raw`?